### PR TITLE
Add logo at browser tab

### LINKED ISSUE #22: [OTHER] Adding favicon.ico
### What would you like to share?

The website is missing a distinctive and recognizable favicon, which can improve its overall appearance and make it easier for users to find and remember.

Goal:

The goal of this issue is to add a favicon to the website to improve its appearance and make it more memorable for users.
Steps:

-Create a square image to use as the favicon, with a recommended size of 16x16 pixels or 32x32 pixels.

-Convert the image to the ICO format, which is the recommended format for favicons.

-Upload the favicon to the root directory of the website.

-Add the following code to the <head> section of the HTML file:

`<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">`

Replace /favicon.ico with the actual location and name of your favicon file.

Expected Outcome:

Once the favicon has been added, it should appear in the browser's tab whenever someone visits the website, making the website more recognizable and memorable for users.

### PR DESCRIPTION
**Pull request by mithindev**
## 🛠️ Fixes Issue

Closes #22 

## 👨‍💻 Changes proposed

The website is missing a distinctive and recognizable favicon, which can improve its overall appearance and make it easier for users to find and remember.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewers

The goal of this issue is to add a favicon to the website to improve its appearance and make it more memorable for users.

✔Create a square image to use as the favicon, with a recommended size of 16x16 pixels or 32x32 pixels.

✔Convert the image to the ICO format, which is the recommended format for favicons.

✔Upload the favicon to the root directory of the website.

✔Add the following code to the <head> section of the HTML file:

`<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">`

✔Replace /favicon.ico with the actual location and name of your favicon file.

### CODE REVIEW

**Comment by anand007s** [comment]
> Can we merge this @Pradumnasaraf

File: public/index.html [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <title>Post Message App</title>
     <meta name="description" content="API">
+    <link rel="shortcut icon" type="image/x-icon" href="/favicon1.ico">
```
Comment (Pradumnasaraf) [inline public/index.html]:
```suggestion
    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico">
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

The icon is not rendering. Looks like the error in naming.

Also, you created PR without getting the issue assigned and approved, which is a violation of CONTRIBUTING guidelines. I am accepting for this time.

**Comment by mithindev** [comment]
I am really sorry, this won't happen again. Thank you so much for reviewing.

**Comment by Pradumnasaraf** [comment]
Hey,

I just had a look the icon design doesn't match the branding. Please use the below image. Also, use `href="favicon.ico"` syntax and remove the `/` 

Untitled design

**Comment by mithindev** [comment]
Ok. I will do it.

**Comment by mithindev** [comment]
Hello @Pradumnasaraf . I have made the changes as you guided me. Thank you so much for your time.

**Review by mithindev — COMMENTED** [review]
Viewed for reviewing!

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Comment by mithindev** [comment]
Thank you so much.

**Pull request merged**